APPLICATION_PROGRESS.md

```diff
@@ -60,6 +60,41 @@ The information in this section is split into subsections in order to conceptual
 
 ---
 
+<details>
+<summary>Automated Market Maker (AMM)</summary>
+
+<h3>Contracts ✅</h3>
+
+- Feature complete for UI integration
+
+<h3>Scripts ❌</h3>
+
+- Two Sway scripts are in progress:
+  1) Swapping assets along a route
+  2) Depositing and adding liquidity atomically
+
+<h3>User Interface</h3>
+
+<h3>Tests</h3>
+
+- <h3>Rust ✅</h3>
+  
+  - The AMM contract has one ignored test because [bytecode root](#https://github.com/FuelLabs/sway/blob/master/sway-lib-std/src/external.sw#L7) cannot be computed via the SDK
```

Comment:
Github changed and I do not have the options to do suggestions etc... the link needs to remove the hashtag at the start